docs/installation/presto-admin-installation.rst

```diff
@@ -5,19 +5,42 @@ Installing Presto Admin
 =======================
 *Prerequisites:* `Python 2.6 or Python 2.7 <https://www.python.org/downloads>`_
 
+To install ``presto-admin``:
+
+Presto Admin is packaged as an offline installer --
+``prestoadmin-<version>-offline.tar.bz2`` -- and as an online
+installer -- ``prestoadmin-<version>-online.tar.bz2``. The offline
+installer includes all of the dependencies for ``presto-admin``, so it
+can be used on a cluster without an outside network connection. The
+offline installer is currently only supported on RedHat Linux 6.x or
+CentOS equivalent.
+
+The online installer downloads all of the dependencies when you call
+``./install-prestoadmin.sh``. You need to use the online installer
```

Comment:
Maybe combine this with the paragraph below:

You must use the online installer for installation of Presto on Amazon EMR and for use on any operating system not listed above.  If you are using ``presto-admin`` on an unsupported operating system, beware that there may be operating system dependencies beyond the installation process, and ``presto-admin`` may not work.